docs/source/getting-started/quickstart.adoc

```diff
@@ -13,7 +13,7 @@ toc::[]
 == Overview
 
 This section contains a brief demo of {project} and of the provisioned OpenShift cluster.
-For details on the usage of {project}, see the xref:../using/managing-minishift.adoc#[Managing {project}] section.
+For details on the usage of {project}, see the xref:../using/basic-commands.adoc#[Managing {project}] section.
```

Comment:
The section was renamed to "Basic Commands", yet the `xref` is still named "Managing {project}" here.

Reply:
Good catch. Let me fix that.